### PR TITLE
Legal docs update

### PR DESCRIPTION
**Pull request by chrisseaton**
Can everyone on the team have a think about this please.

### CODE REVIEW

File: mx.truffleruby/suite.py [hunk 1]
```diff
@@ -81,7 +86,9 @@
             },
             "sha1": "3ba3cb50e9a0b87b645cd412a8f7b7296c314aa1",
             "sourceSha1": "2fdcca5bd2c629c8c120d46cc1ed325c994ea3eb",
-            "licence": "MIT"
+            "licence": [
```
Comment (bjfish) [inline mx.truffleruby/suite.py]: I noticed some of these are `licence` and some are `license`

Comment (chrisseaton) [reply mx.truffleruby/suite.py]: `mx` itself also uses both :(

Comment (chrisseaton) [reply mx.truffleruby/suite.py]: Fixed.

File: mx.truffleruby/suite.py [hunk 2]
```diff
@@ -275,7 +327,13 @@
                 "truffleruby-doc",
             ],
             "description": "TruffleRuby libraries, documentation, bin directory",
-            "license": ["EPL-1.0", "MIT", "BSD-simplified", "GPLv2", "LGPLv21", "zlib"],
+            "licenses": [
+                "EPL-1.0",          # JRuby (we're choosing EPL out of EPL,GPL,LGPL)
+                "MIT",              # minitest, did_you_mean, rake
+                "BSD-simplified",   # MRI
+                "BSD-new",          # Rubinius, FFI
```
Comment (eregon) [inline mx.truffleruby/suite.py]: and RubySL

**Pull request merged**